src/main/scala/IOService/Http4sTestServer.scala

```diff
@@ -0,0 +1,109 @@
+package IOService
+import cats._
+import cats.effect._
+import cats.implicits._
+import org.http4s.circe._
+import org.http4s._
+import io.circe.generic.auto._
+import io.circe.syntax._
+import org.http4s.dsl._
+import org.http4s.implicits._
+import org.http4s.server.blaze.BlazeServerBuilder
+import cats.Monad
+import cats.effect.unsafe.implicits.global
+import cats.effect.{Concurrent, ExitCode, IO, IOApp}
+import org.http4s.server.Router
+import org.scalacheck.Gen
+import poker.Deck.startingDeck
+import poker.{Card, Street}
+import poker.Street._
+
+import scala.collection.mutable
+
+object Http4sTestServer extends IOApp {
+
+  /*
+  - Get all the players and cards of current Street - the board state
+  - Post - deal the hole cards -  preflop state
+   */
+
+  // Request -> F[Option[Response]]
+  // HttpRoutes[F] ^^^
+
+  // Get /flop
+  // GET /turn
+  // GET /river
+  // GET /players
+
+  // ??? Need a DB/state
+
+  def genPreFlopBoard(numPlayers: Int): Gen[Preflop] =
+    startingDeck.dealHoleCards(numPlayers).unsafeRunSync()
```

Comment:
What makes `unsafeRunSync` necessary?

Reply:
let me check.  So iiuc, in the branch this is based off of` dealHoleCards` returns an `IO`.  (My other draft pr is trying to return an `F`, but that is not merged. ) 
To be honest... now I'm not sure why I have  ` dealHoleCards` in an `IO` I think it is pure.  My thinking now is `shuffle` needs to return an `F` because it's doing side effects, but not ` dealHoleCards`.   

So I think when I get the Shuffle PR correct and merged, I'll be able to get rid of this` unsafeRunSync()`

Reply:
Your HTTP routes are basically in `IO` anyway though right?

Reply:
In general, the idea of using `Gen` to create random data for this scenario makes sense. What doesn't make sense to me here is: shuffling is the randomization, right? So you don't need the features of `Gen`, just `IO`?

Reply:
I think your point is getting to the main part of the exercise.  (I have a family function, but I think I understand your point.)  I can use map up to a certain point.  but when I reach the actual HttpRoutes... might flail a bit. bbl